src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -220,6 +220,16 @@ std::vector<std::string> disabledTestPatterns() {
     retVector.emplace_back(R"(smoke_NegativeQuantizedMatMulMultiplyFusion.*)");
     // int8 specific
     retVector.emplace_back(R"(smoke_Quantized.*)");
+    retVector.emplace_back(R"(smoke_Conv_Sum_Broadcast_INT8/ConvSumInPlaceTestInt8.*)");
+    // ACL does not support blocked memory formats
+    retVector.emplace_back(R"(.*(in|out)Fmts=nC(w|hw|dhw)(8|16)c.*)");
+    //FQ, Add and Swish post-ops are not supported by ACL
+    retVector.emplace_back(R"(smoke_Conv_Sum_Broadcast.*Fused=.*(FakeQuantize|Add).*)");
```

Comment:
Unsupported subgraph should just be backend specific
applicable for all the skips

Reply:
Removed these skips and moved tests with blocked memory format to x64 folder